ci/build_setup.sh

```diff
@@ -62,7 +64,8 @@ export BAZEL="bazel"
 # Not sandboxing, since non-privileged Docker can't do nested namespaces.
 BAZEL_OPTIONS="--package_path %workspace%:${ENVOY_SRCDIR}"
 export BAZEL_QUERY_OPTIONS="${BAZEL_OPTIONS}"
-export BAZEL_BUILD_OPTIONS="--strategy=Genrule=standalone --spawn_strategy=standalone \
+export BAZEL_BUILD_OPTIONS="--action_env=PATH=/bin:/usr/bin:/usr/lib/llvm-7/bin --linkopt=-fuse-ld=lld \
```

Comment:
If we are going to link the release build with lld I think we should probably add a release note? I'm not sure that it really matters but it seems like something we should note perhaps? WDYT?

Reply:
yeah sure, added to version_history along with compiler update.